### ISSUE TITLE
Iosevka Term SS09 16.2.0 looks like a different font from 3.4.3

### ISSUE THREAD

**Issue by step-** [be5invis/Iosevka#1420]
This is a continuation of #1419, reopening to show you new evidence. Thank you for your patience.

In https://github.com/be5invis/Iosevka/issues/1419#issuecomment-1249955224 @be5invis wrote:

> This is what it supposed to look like.

I see. Indeed the font looked like that in my terminal when I upgraded from 3.4.3 to 16.2.0.  However, I still think there's been a big change in the look of Term SS09 somewhere along the chain of releases. Before upgrading to 16.2.0 I was using 3.4.3 TTF - which is now installed in my system. Take a look at what I see.

![iosevka-20220917](https://user-images.githubusercontent.com/1414102/190843281-bbe11e32-7fbf-4418-87a6-68551c76ea08.png)

The very bottom line shows the command I used to start urxvt.  The lines above sit in the urxvt terminal I just started. The purple line is italics font. The gray line is regular font.  You can see that the characters look like the picture I posted for 16.0.1, unlike the characters in the 16.0.2 picture.  In 2020 I had also saved the 3.4.3 font list PDF file, and the page for Term SS09 is attached.  You can see that the test characters look like the picture in 16.0.1, unlike the characters in 16.0.2.

[Release Iosevka, version 3.4.3.pdf](https://github.com/be5invis/Iosevka/files/9591071/Release.Iosevka.version.3.4.3.pdf)

Perhaps this change didn't happen between 16.0.1 and 16.0.2 as I thought.  When I upgraded to 16.2.0 [sic] the font in my terminal looked different from before.  I can think of three possible explanations: (a) I had installed the wrong font; (b) I had installed a TCC bundle, whereas earlier I had been using straight TTF 3.4.3, and maybe my system isn't properly configured for TCC; (c) the font changed indeed.  I feel I can rule out (a) because I only download Term SS09 TCC and TTF. I can't rule out (b) but I also tested TTF 16.2.0 and saw no improvement. So I filed this issue because I thought I had enough evidence of (c).

_Originally posted by @step- in https://github.com/be5invis/Iosevka/issues/1419#issuecomment-1250011898_

**Comment by alexeyten** [comment]
I guess it was changed in #1065, #1378, #1379 and maybe some other issues/commits.

**Comment by be5invis** [comment]
They are changed to better mimic Source Code Pro. Maybe you will need a custom build to get the variant you like.

**Comment by step-** [comment]
@alexeyten, thanks for providing those links that bring in some needed historical context. @be5invis wrote:

> Maybe you will need a custom build to get the variant you like.

You're right, I reviewed all variants and I will stick with SS09 - even with its current form - but I definitely need to change the digits, especially that "1", which looks terrible IMO, but I digress.  I was really holding back to doing a custom build because I remembered a lot of complications from my last attempt in 2019... but what a gigantic surprise in 2022, the [customizer tool](https://typeof.net/Iosevka/customizer) is so user-oriented and easy to use! This whole site is a work of genius and love, thanks!

P.S. Is it coincidental that @alexeyten's avatar is a cup of coffee and @be5invis's a doughnut? :)
